engine/execution/messages.go

```diff
@@ -53,38 +137,30 @@ func NewEmptyComputationResult(
 	}
 }
 
-func (cr ComputationResult) transactionResultsByCollectionIndex(colIndex int) []flow.TransactionResult {
-	var startTxnIndex int
-	if colIndex > 0 {
-		startTxnIndex = cr.TransactionResultIndex[colIndex-1]
-	}
-	endTxnIndex := cr.TransactionResultIndex[colIndex]
-	return cr.TransactionResults[startTxnIndex:endTxnIndex]
-}
-
-func (cr *ComputationResult) CollectionResult(colIndex int) *ColResSnapshot {
-	if colIndex < 0 && colIndex > len(cr.CompleteCollections) {
+func (cr *AttestationResults) AttestedCollection(colIndex int) *AttestedColSnapshot {
+	if colIndex < 0 && colIndex > cr.AttestedColCounter {
 		return nil
 	}
-	return &ColResSnapshot{
-		blockHeader: cr.Block.Header,
-		collection: &flow.Collection{
-			Transactions: cr.CollectionAt(colIndex).Transactions,
-		},
-		updatedRegisters:   cr.StateSnapshots[colIndex].UpdatedRegisters(),
-		readRegisterIDs:    cr.StateSnapshots[colIndex].ReadRegisterIDs(),
-		emittedEvents:      cr.Events[colIndex],
-		transactionResults: cr.transactionResultsByCollectionIndex(colIndex),
+	return &AttestedColSnapshot{
+		startStateCommit: cr.Chunks[colIndex].StartState,
+		endStateCommit:   cr.Chunks[colIndex].EndState,
+		stateProof:       cr.ChunkDataPacks[colIndex].Proof,
+		eventCommit:      cr.EventsHashes[colIndex],
 	}
 }
 
 type ColResSnapshot struct {
```

Comment:
maybe rename to CollectionExecutionResults

(in general, use full name please.  short hand is less skimmable)

Reply:
renamed at [b8ccd7b](https://github.com/onflow/flow-go/pull/4078/commits/b8ccd7b4712e45aa01942322001a63e01c3b9342)